pre_commit/languages/swift.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+
+import os
+
+from pre_commit.languages import helpers
+from pre_commit.util import clean_path_on_failure
+from pre_commit.xargs import xargs
+
+ENVIRONMENT_DIR = 'swift_env'
+BUILD_DIR = '.build'
+BUILD_CONFIG = 'release'
+
+
+def install_environment(
+        repo_cmd_runner,
+        version='default',
+        additional_dependencies=(),
+):
+    directory = repo_cmd_runner.path(helpers.environment_dir(
+        ENVIRONMENT_DIR, 'default',
```

Comment:
I would assert that the version is default since no other version is supported, there's prior art in the docker language